.gitlab-ci.yml

```diff
@@ -165,3 +170,32 @@ Release GCC Module:
     CMAKE_BUILD_TYPE: "Release"
     MODULE_BUILD: "yes"
   extends: .build
+
+Debug GPU Ubuntu:
+  retry:
+    max: 2
+  variables:
+    CC: "gcc"
+    CXX: "g++"
+    CCACHE_DIR: "${CI_PROJECT_DIR}/ccache"
+    CXXFLAGS: "-Wall -Werror"
```

Comment:
This is the same in both cases, so leave this property with `.build_script`.

Reply:
done